types/ramda/index.d.ts

```diff
@@ -2331,11 +2331,58 @@ export function propIs<C extends new (...args: any[]) => any>(
  * If the given, non-null object has an own property with the specified name, returns the value of that property.
  * Otherwise returns the provided default value.
  */
-export function propOr<T, U>(val: T, __: Placeholder, obj: U): <V>(p: string) => V;
-export function propOr<U>(__: Placeholder, p: string, obj: U): <T, V>(val: T) => V;
-export function propOr<T, U, V>(val: T, p: string, obj: U): V;
-export function propOr<T>(val: T, p: string): <U, V>(obj: U) => V;
-export function propOr<T>(val: T): <U, V>(p: string, obj: U) => V;
+export function propOr<O extends object>(
```

Comment:
Placeholders introduce too much complexity with little gains, so I don't think it's a good idea to continue compatibility with placeholders.

Reply:
I also wouldn't add those typings until there is a decision wether or not we want to continue support